### PR TITLE
chore: minor updates bump

### PR DESCRIPTION
**Pull request by aljo242**
## 1. Summary
- update minor dependencies
- update `setup-go` action from `v3` to `v4`
- update `sdk` to `v0.46.12`
- update `gofumpt` and run `make format`

### CODE REVIEW

File: .github/workflows/buildtest.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This is a basic workflow that is manually triggered
+4# This is a basic workflow that is manually triggered
```
Comment (joe-bowman) [inline .github/workflows/buildtest.yaml:1]: I suspect this will break the flow?

Comment (aljo242) [reply .github/workflows/buildtest.yaml:1]: Lol yes it will

Comment (joe-bowman) [inline .github/workflows/buildtest.yaml:1]:
```suggestion
# This is a basic workflow that is manually triggered
```

**Review by joe-bowman — APPROVED** [review]
Resolved issue. Lgtm

**Review by ajansari95 — APPROVED** [review]
Lgtm

**Comment by ajansari95** [comment]
Waiting for checks

**Pull request merged**